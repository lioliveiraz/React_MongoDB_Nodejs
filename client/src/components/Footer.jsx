import React from 'react';

function Footer(props) {
    return (
        <div style={{background:"green"}}>
                Footer
        </div>
    );
}

export default Footer; 