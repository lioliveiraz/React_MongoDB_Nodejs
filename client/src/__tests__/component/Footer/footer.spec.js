import Footer from "../../../components/Footer";
import { shallow } from "enzyme";

describe("Footer", () => {
  it("should match Snapshot", () => {
    let wrapper = shallow(<Footer />);

    expect(wrapper).toMatchSnapshot();
    expect(wrapper.html).toMatchSnapshot();
  });
});
