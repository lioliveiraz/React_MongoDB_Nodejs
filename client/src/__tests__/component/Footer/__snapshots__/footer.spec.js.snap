// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`  1`] = `ShallowWrapper {}`;

exports[`  2`] = `[Function]`;

exports[`Footer should match Snapshot 1`] = `ShallowWrapper {}`;

exports[`Footer should match Snapshot 2`] = `[Function]`;
